python/ray/experimental/data/read_api.py

```diff
@@ -146,6 +146,7 @@ def read_parquet(paths: Union[str, List[str]],
         Dataset holding Arrow records read from the specified paths.
     """
     import pyarrow.parquet as pq
+    import numpy as np
```

Comment:
are we ok to take numpy as a dependency?  i did something similar here: https://github.com/ericl/ray/pull/16/files#diff-651d4a994917fc5158a5a2dc6b427ac243039b30a606815b82a27c0a7c0103ddR228

Reply:
I'm not too worried about the NumPy dependency, but if it proves to be concerning, we can definitely create a utility that mimics `np.array_split` and use that instead!

Reply:
Numpy is already a core dep of ray, you can just top-level import.